docs/2 Architecture/3 Standard Modules.md

```diff
@@ -1,4 +1,13 @@
-# Cosmos SDK Basic Modules
+# What are modules
+
+In the Cosmos SDK modules are the basis for the logic of your blockchain. Each module serves specific purposes and functions. The Cosmos SDK offers a variety of native modules to make a blockchain work. These modules handle authentification for users, token transfers, governmental functions, staking of tokens, supply of tokens and many more.
+
+If you want to change the default functionality of a module or just change certain hardcoded parameter, you can fork a module and change it, therefore owning your own logic for your blockchain. While forking and editing a module should be done carefully, this approach marks the Cosmos SDK as especially powerful, as you can experiment with different parameters as the standard implementation suggests.
+
+Modules do not need to be created by a specific company or invididual. They can be created by anyone and offered for general use to the public. Although there do exist standards that projects look into before integrating a module to their blockchain. It is recommended that a module has understandable specification, handles one thing good and is well tested - optimally battle-tested on a live blockchain.
+When growing more complex, sometimes it makes more sense to have two modules instead of one module trying to "solve-it-all", this consideration can make it more attractive for other projects to use a module on their blockchain project.
```

Comment:
```suggestion
When growing more complex, sometimes it makes more sense to have two modules instead of one module trying to "solve-it-all", this consideration can make it more attractive for other projects to use a module in their blockchain project.
```